### PR TITLE
[#2551] simplified profile path in cse_run.log

### PR DESCRIPTION
**Pull request by david6969xin**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install -Pit` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: foundations/foundation-config/src/test/java/org/apache/servicecomb/config/archaius/sources/TestMicroserviceConfigLoader.java [hunk 1]
```diff
@@ -102,4 +102,15 @@ public void testLoadNotExistYaml() throws IOException {
       Assert.assertTrue(true);
     }
   }
+
+  @Test
+  public void testConvertUrl() {
+    String fileUrlOne = "file:/C:/gitSpace/servicecomb-java-chassis/transports"
+        + "/transport-highway/target/classes/microservice.yaml";
+    assertEquals("g/s/t/classes/microservice.yaml", ConfigModel.convertUrl(fileUrlOne));
```
Comment (liubao68) [inline foundations/foundation-config/src/test/java/org/apache/servicecomb/config/archaius/sources/TestMicroserviceConfigLoader.java:110]: consider windows `D:\a.txt`

File: foundations/foundation-config/src/main/java/org/apache/servicecomb/config/archaius/sources/ConfigModel.java [hunk 2]
```diff
@@ -54,6 +56,30 @@ public void setConfig(Map<String, Object> config) {
...
+
+  private static String InitialsInfo(String target) {
+    StringBuilder fileDetailString = new StringBuilder();
+    while (target.contains("/")) {
+      String temp = target.substring(target.indexOf("/"), target.indexOf("/") + 2);
+      fileDetailString.append(temp);
+      target = StringUtils.remove(target, temp);
+    }
+    return fileDetailString.toString();
   }
```
Comment (liubao68) [inline foundations/foundation-config/src/main/java/org/apache/servicecomb/config/archaius/sources/ConfigModel.java:84]: add a util method to org.apache.servicecomb.foundation.common.utils.IOUtils

Comment (david6969xin) [reply foundations/foundation-config/src/main/java/org/apache/servicecomb/config/archaius/sources/ConfigModel.java:84]: ok

Comment (liubao68) [inline foundations/foundation-config/src/main/java/org/apache/servicecomb/config/archaius/sources/ConfigModel.java:84]: This algorithm is a bit complicated. e.gs

String convertString(String orgin) {
    char[] seq = origin.length() > 60 ? origin.subString(0, 60) : origin.toChars();
    if(seq.lenght < 20) return ...

    StringBuilder sb = new StringBuilder();
    boolean m = false;
    for(int i = 0; i<seq.length - 20; i++) {
       if(c == '/' || c=='\\') {
...
       }
    }
}

File: foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/utils/IOUtils.java [hunk 3]
```diff
@@ -18,9 +18,37 @@
...
+import org.apache.commons.lang3.StringUtils;
+
 public class IOUtils {
   @SuppressWarnings("deprecation")
   public static void closeQuietly(final Closeable closeable) {
     org.apache.commons.io.IOUtils.closeQuietly(closeable);
   }
+
+  public static String convertString(String origin) {
+    return convertString(origin,System.getProperty("file.separator").charAt(0));
```
Comment (liubao68) [inline foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/utils/IOUtils.java:30]: extract a constant for this

File: foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/utils/IOUtils.java [hunk 4]
```diff
@@ -18,9 +18,37 @@
...
+  }
+
+  public static String convertString(String origin, char separator) {
+    if (StringUtils.isEmpty(origin)) {
+      return "";
```
Comment (liubao68) [inline foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/utils/IOUtils.java:35]: return origin

**Review by liubao68 — APPROVED** [review]

**Pull request merged**